.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,23 @@
+name: C/C++ CI
+
+on: [push]
```

Comment:
Actually, would you mind changing on push to `on: [push, pull_request]` ?

Reply:
done!